src/eliminate_pad.cpp

```diff
@@ -55,21 +63,30 @@ void eliminate_pad::update_pooling(const instruction_ref& input,
                                    module& p) const
```

Comment:
The update_pooling and update_op can be combined with a template now. The reason that it is a separate function is the averagepooling check at line 78.